root/scripts/download.bash

```diff
@@ -538,7 +538,7 @@ DownloadVideos () {
 			fi
 			youtubeaveragerating="$(echo "$youtubedata" | jq -r '.average_rating')"
 			videoalbum="$(echo "$youtubedata" | jq -r '.album')"
-			sanatizedvideodisambiguation=""
+			sanitizedvideodisambiguation=""
```

Comment:
If changing this, you need to make sure you change the other places...

Reply:
I changed all of them, isn't it?

Reply:
Don't think so, considering this is a single change, and usually its in more than one place...